Documentation/building.md

```diff
@@ -22,10 +22,13 @@ Then you should be able to set the ARCH as above
 ## Building manually
 
 1. Make sure you have required dependencies installed on your machine.
-    * On Ubuntu, run `sudo apt-get install linux-libc-dev golang gcc`.
+    * On Ubuntu, run `sudo apt-get install linux-libc-dev golang gcc`. 
+      If the golang version installed is not 1.7 or higher. Download the newest golang and install manully.
```

Comment:
Typo: manually

Reply:
Yeah. You can just edit it or another PR from me?